easy-connect.cpp

```diff
@@ -195,11 +195,13 @@ NetworkInterface* easy_connect(bool log_messages) {
 #endif // EASY_CONNECT_WIFI
 
     /// This should be removed once mbedOS supports proper dual-stack
+    if (log_messages) {
 #if defined (EASY_CONNECT_MESH) || (MBED_CONF_LWIP_IPV6_ENABLED==true)
-    printf("[EasyConnect] IPv6 mode\n");
+        printf("[EasyConnect] IPv6 mode\n");
```

Comment:
Is there a tab/spaces issues here or not?

Reply:
I added an indentation level but if Github's interface is correct, it should be consistent with the previous level.

Reply:
No, there isn't, all OK.